fbw-common/src/systems/shared/src/checklists/index.ts

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2023-2024 FlyByWire Simulations
+// SPDX-License-Identifier: GPL-3.0
+
+import { getAircraftType } from '@flybywiresim/fbw-sdk';
+import {
+    afterLandingChecklistA32NX,
+    afterStartChecklistA32NX,
+    approachChecklistA32NX,
+    beforeStartChecklistA32NX,
+    cockpitPreparationChecklistA32NX,
+    landingChecklistA32NX,
+    lineUpChecklistA32NX,
+    parkingChecklistA32NX,
+    securingAircraftChecklistA32NX,
+    taxiChecklistA32NX,
+} from './Lists_A32NX/A32NX_Checklists';
+import { ChecklistDefinition } from '../../../instruments/src/EFB/Checklists/Checklists';
+import {
+    afterLandingChecklistA380X,
+    afterStartChecklistA380X,
+    afterTakeoffChecklistA380X,
+    approachChecklistA380X,
+    beforeStartChecklistA380X,
+    beforeTakeoffChecklistA380X,
+    landingChecklistA380X,
+    parkingChecklistA380X,
+    securingAircraftChecklistA380X,
+} from './Lists_A380X/A380X_Checklists';
+
+export const CHECKLISTS_A32NX: ChecklistDefinition[] = [
+    cockpitPreparationChecklistA32NX,
+    beforeStartChecklistA32NX,
+    afterStartChecklistA32NX,
+    taxiChecklistA32NX,
+    lineUpChecklistA32NX,
+    approachChecklistA32NX,
+    landingChecklistA32NX,
+    afterLandingChecklistA32NX,
+    parkingChecklistA32NX,
+    securingAircraftChecklistA32NX,
+];
+
+export const CHECKLISTS_A380X: ChecklistDefinition[] = [
+    beforeStartChecklistA380X,
+    afterStartChecklistA380X,
+    beforeTakeoffChecklistA380X,
+    afterTakeoffChecklistA380X,
+    approachChecklistA380X,
+    landingChecklistA380X,
+    afterLandingChecklistA380X,
+    parkingChecklistA380X,
+    securingAircraftChecklistA380X,
+];
```

Comment:
This uses a ton of extra memory for no reason. We'd be far better off to either use a plugin that can provide the checklists, or have a separate instrument for the two planes and pass config in through that.

Reply:
Don't think it's a ton as these are simple and small data structures. But I get the point. 
But sending all the aircraft differences as parameters is also not a good solution. 
Would be better to do it like we do it in cpp or as you said with a plugin system. 
But maybe something for v4